include/jemalloc/internal/thread_event.h

```diff
@@ -81,6 +80,58 @@ ITERATE_OVER_ALL_COUNTERS
  */
 #undef E
 
+/*
+ * Two malloc fastpath getters -- use the unsafe getters since tsd may not be
+ * nominal, in which case the fast_threshold will be set to 0.  This allows
+ * checking for events and tsd non-nominal in a single branch.
+ */
+JEMALLOC_ALWAYS_INLINE uint64_t
+thread_allocated_get_malloc_fastpath(tsd_t *tsd) {
```

Comment:
nit: also get rid of the `get` to be consistent with `thread_allocated_next_event_malloc_fastpath()`?

Reply:
Good point. Just updated.